scala/src/main/scala/smile/clustering/distances/Binary/HammingDistance.scala

```diff
@@ -0,0 +1,15 @@
+package smile.clustering.distances
```

Comment:
Distance classes should be in the package `smile.math.distance`. We have many distance classes already defined there, e.g. HammingDistance.